### DISCUSSION TITLE
Uninstall ClockEleven

### DISCUSSION THREAD

**Discussion by fmarto** [martinet101/ElevenClock#542] [Q&A] [answered]
How can I uninstall ClockEleven?

Thanks.

**Comment by Masamune3210** [comment, marked as answer]
ElevenClock, and the same way you uninstall any other program

**Reply by marticliment** [reply]
You can additionally remove the .elevenclock folder placed in your user directory

**Reply by fmarto** [reply]
I know how to uninstall any other program but when it comes to ElevenClock, the "Uninstall" option is disabled.

**Reply by marticliment** [reply]
Weird... Try reinstalling and then uninstalling...

**Reply by TechStudent10** [reply]
@fmarto That's weird. Where are you uninstalling the program from? Control Panel or Windows Settings?

**Reply by BionicHobbit** [reply]
>fmarto
I know how to uninstall any other program but when it comes to ElevenClock, the "Uninstall" option is disabled.


I'm having the same problem...it's telling me it can't find the uninstaller.  Wth.

**Reply by marticliment** [reply]
Then please try reinstalling ElevenClock from the Microsoft Store and then attempt to uninstall it

**Comment by marticliment** [comment]
@fmarto, is someting not working?

**Comment by Masamune3210** [comment]
Worst case you can also use something like BCUninstaller to force windows to call the uninstaller
